src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, ChangeEvent, useEffect } from "react";
```

Comment:
You shouldn't have part of the project with single quote and some with double quotes.  I recommend single and to keep it consistent across all files, using a prettify or lint tool